specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json

```diff
@@ -8,7 +8,6 @@
     "description": "This Swagger was generated by the API Framework.",
     "version": "2018-07-01"
   },
-  "host": "management.azure.com",
   "consumes": [
```

Comment:
>consumes [](start = 3, length = 8)

what is the reason for removing 'host' ?

Reply:
IMO it is required

---
In reply to: [387329551](https://github.com/Azure/azure-rest-api-specs/pull/8297#discussion_r387329551) [](ancestors = 387329551)

Reply:
same feedback for other files

---
In reply to: [387329591](https://github.com/Azure/azure-rest-api-specs/pull/8297#discussion_r387329591) [](ancestors = 387329591,387329551)